.github/workflows/regression.yml

```diff
@@ -11,13 +11,31 @@ on:
     paths-ignore:
       - "**/*.md"
       - "LICENSE"
-
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Wonder what do we expect from this. Will things change between last merge and every day run?

Reply:
removed